apps.yml

```diff
@@ -1,3 +1,27 @@
+# NOTE: authorized_groups and authorized_users control two things independently:
+#
+# 1. Does the SSO Dashboard display a tile to the user for this application?
+#
+#        No, if display: false.
+#            No SSO users can see the tile.
+#
+#        Maybe, if display: true.
+#            If groups/users are defined,
+#                The user must be a member of them to see the tile.
+#            Otherwise,
+#                All SSO users (including the general public) can see the tile.
+#
+# 2. Does Auth0 restrict SSO access to this application by group or user?
+#
+#        No, if client_id is missing.
+#            All SSO users (including the general public) can access the app.
+#
+#        Maybe, if client_id is defined.
+#            If groups/users are defined,
+#                The user must be a member of them to access the application.
```

Comment:
The user must either be in the `authorized_users` list or be a member of at least one of the groups in the `authorized_groups` list. Either of these conditions satisfy the requirement.

Reply:
Yeah, my wording is imperfect, but "the user is in one of the groups or the user is in the users list" starts to get weird, and it balloons from there. One of my criteria was 'fits into an 80x25 window' so that everyone reads it when they open the file in an editors, at least if they start at the top, which many do. If you feel there is an alternate interpretation that is materially problematic I'm happy to take more PRs here.